packages/cli/src/commands/freeze.js

```diff
@@ -5,7 +5,7 @@ import runWithTruffle from '../utils/runWithTruffle'
 
 const name = 'freeze'
 const signature = name
-const description = 'freeze current release version of your stdlib project'
+const description = 'freeze current release version of your package project'
```

Comment:
```suggestion
const description = 'freeze current release version of your published project'
```

Now we can freeze any kind of project, as long as its published.